harness/determined/integrations/huggingface/hf_callback.py

```diff
@@ -297,6 +301,7 @@ def get_metric_type(d):
             return TRAIN_AVG
         else:
             return TRAIN
+    return TRAIN
```

Comment:
For the empty `dict` case, I guess?  I still need to understand this function better. Guess HF ensure homogeneous keys in `d`? Like, they all start with the same prefix.

Reply:
That's what I assume as well. This is special sauce for the internals of huggingface. I think we are just assuming that it does indeed prefix like this